Log to /var/log/nntsc/ rather than /tmp/.
--- a/nntsc
+++ b/nntsc
@@ -58,7 +58,7 @@
     def __init__(self, listen_port, config, backgrounded, exportonly, 
             querytimeout):
         if backgrounded:
-            logger.createLogger(True, "/tmp/nntsc.log", "NNTSC")
+            logger.createLogger(True, "/var/log/nntsc/nntsc.log", "NNTSC")
         self.config = config
 
         # Work out which modules to blacklist
